client/components/mma/switch/review/SwitchReview.tsx

```diff
@@ -82,23 +82,28 @@ const PaymentDetails = (props: { subscription: Subscription }) => {
 			)}
 			{subscription.payPalEmail && (
 				<>
-					<span css={truncateCss}>
+					<span css={truncateCss} data-qm-masking="blocklist">
+						{' '}
 						{getObfuscatedPayPalId(subscription.payPalEmail)}
 					</span>
 					<PaypalLogo />
 				</>
 			)}
 			{subscription.mandate && (
 				<>
-					account ending{' '}
-					{subscription.mandate.accountNumber.slice(-3)}
+					<span data-qm-masking="blocklist">
+						account ending{' '}
+						{subscription.mandate.accountNumber.slice(-3)}
+					</span>
```

Comment:
Same as the above — the extra `span` and attribute probably aren't needed.

Reply:
Good call - removed